crates/state/src/lib.rs

```diff
@@ -1,5 +1,5 @@
-#![allow(dead_code)] // TODO: remove once the bridge state `sanity_check` fn is used.
-#![feature(is_sorted)] // TODO: switch to using crate
+#![allow(stable_features)] // FIX: this is needed for sp1 toolchain.
+#![feature(is_sorted, is_none_or)]
```

Comment:
Could you add a comment as to why this one is needed to?

Reply:
The problem is that this will conflict with `main`: https://github.com/alpenlabs/strata/blob/5008f3ddc74d14580fc40a443710b1e8352b154b/crates/state/src/lib.rs#L1-L2

Reply:
Okay then. We can leave this as is for now.